lms/djangoapps/philu_api/helpers.py

```diff
@@ -13,3 +15,28 @@ def get_course_custom_settings(course_key):
         course_key = CourseKey.from_string(course_key)
 
     return CustomSettings.objects.filter(id=course_key).first()
+
+
+def get_social_sharing_urls(course_url, meta_tags):
+    social_sharing_urls = {}
+    utm_params = meta_tags['utm_params'].copy()
+    share_url = '{}?{}'.format(course_url, urlencode(utm_params))
+
+    # Facebook
+    facebook_share_url = 'https://www.facebook.com/sharer/sharer.php'
+    share_url = add_or_replace_parameter(share_url, 'utm_source', 'Facebook')
+    social_sharing_urls['facebook'] = add_or_replace_parameter(facebook_share_url, 'u', share_url)
+
+    # LinkedIn
```

Comment:
Let's try to remove duplication by making a method which takes a param `Facebook`, `LinkedIn` or `Twitter` and send us the social sharing url.